packages/sitecore-jss-nextjs-editing-host/src/config.test.ts

```diff
@@ -0,0 +1,91 @@
+import chai = require('chai');
+import chaiString = require('chai-string');
```

Comment:
@ambrauer You can use `import` instead of `require`

Reply:
@sc-illiakovalenko Good catch, switched.